widget/alsabar.lua

```diff
@@ -51,20 +55,23 @@ local function factory(args)
         alsabar.notification_preset.font = "Monospace 10"
     end
 
-    local format_cmd = string.format("%s get %s", alsabar.cmd, alsabar.channel)
+    alsabar.cmds.get_cmd = string.format("%s get %s", alsabar.alsa_bin, alsabar.channel)
+    alsabar.cmds.inc_cmd = string.format("%s sset %s %s+", alsabar.alsa_bin, alsabar.channel, step)
+    alsabar.cmds.dec_cmd = string.format("%s sset %s %s-" , alsabar.alsa_bin, alsabar.channel, step)
+    alsabar.cmds.tog_cmd = string.format("%s sset %s toggle", alsabar.alsa_bin, alsabar.channel)
 
```

Comment:
These commands are just useless here if the user does not need them.

But we can add them to a theming/configuring submodule. https://github.com/lcpz/lain/issues/317